src/uwtools/resources/rocoto.jsonschema

```diff
@@ -114,44 +270,50 @@
       ],
       "anyOf": [
         {
-            "not": {
-                "required": ["exclusive"]
-            }
-        },
-        {
-            "not": {
-                "required": ["shared"]
```

Comment:
I'm not following the logic on why the logic here is removed. I see below it's mainly just reformatting.

Reply:
Hrm, could be a conflict resolution error, let me recheck everything

Reply:
Oh weird, `jq -r` removed it... I'm overriding it for now - any idea why it would do that? @maddenp-noaa ?

Reply:
Yes: You have (or had), two `anyOf` keys at the same level. That's not possible in a JSON object (as it would not be possible in a Python `dict`), so there's something wrong with the schema -- if we've got a schema with two identical keys at the same level, then the schema is incorrectly constructed. `jq` did the right thing by removing one of them. In general, "last in wins" with these things; Python behaves the same:
```
>>> d = {"a": 1, "a": 2, "a": 3}
>>> d
{'a': 3}
```

Reply:
Hrm, I'm struggling a bit to solve this one - @NaureenBharwaniNOAA 's implication use here from 381 is excellent otherwise. Can we stash these within `dependentSchemas`? ChatGPT only got confused when I asked it, of course.